### PR TITLE
#124 FemaleCondom clean up.

### PR DESCRIPTION
**Pull request by LeonelADiaz**
#124 Clean up and merged into one file.

### CODE REVIEW

File: src/Components/FemaleCondom.jsx [hunk 1]
```diff
@@ -2,11 +2,191 @@ import React, { useState, useEffect } from "react";
 import { TouchableHighlight, Text, View, ScrollView, Image } from "react-native";
 import appStyles from "./AppStyles";
 import SelectionButton from "./SelectionButton";
+<<<<<<< HEAD
```
Comment (edumorlom) [inline src/Components/FemaleCondom.jsx:5]: I think u forgot to fix this merging conflict :)

**Review by edumorlom — CHANGES_REQUESTED** [review]

File: src/Components/FemaleCondom.jsx (hunk 1 above)
Comment (LeonelADiaz) [reply src/Components/FemaleCondom.jsx]: Sorry about that, should be good now!

**Review by edumorlom — APPROVED** [review]
Looks good, was tested correct?

**Comment by LeonelADiaz** [comment]
> 
> 
> Looks good, was tested correct?

Yup. Works exactly as before, just now in one file. 

There was old code there commented out that attempted to solve an onBack() issue. When you press back on either the Condom Steps or Do/Dont screens, they take you back to the Learn screen instead of the Female Condom screen. I'll open this up as a new issue.

**Pull request merged**